types/angular-aria/index.d.ts

```diff
@@ -0,0 +1,20 @@
+// Type definitions for angular-aria 1.7
+// Project: http://angularjs.org, https://docs.angularjs.org/api/ngAria
+// Definitions by: Chives <https://github.com/chivesrs>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.3
+
+import * as angular from 'angular';
+
+declare module 'angular' {
+    namespace aria {
+        type IAriaAttribute = 'ariaHidden'|'ariaChecked'|'ariaReadonly'|'ariaDisabled'|'ariaRequired'|'ariaInvalid'|'ariaValue'|'tabindex'|'bindKeydown'|'bindRoleForClick';
+
+        /**
+         * $aria service.
+         */
+        interface IAriaService {
+            config(attribute: IAriaAttribute): boolean;
```

Comment:
So I think this is wrong, the ngAria, as per author intent, provides a configuration object (provider) to configure supported attributes:
https://gist.github.com/marcysutton/8e4f6e51d82cb36710c4#example
This provider comes with `config` method that accepts object:
https://github.com/angular/angular.js/blob/master/src/ngAria/aria.js#L97
the `$ariaService` is for internal purpose only and not exposed as public api I think:
https://docs.angularjs.org/api/ngAria/service/$aria

Reply:
$aria is a public API, the doc you see is broken.
See https://github.com/angular/angular.js/blob/master/src/ngAria/aria.js#L160 for docs.

Reply:
I'll see if I can get the docs functional, but you can see on the main ngAria page that $aria is meant to be a public service.

Yes, this PR doesn't add typings for $ariaProvider at all. That's because my code doesn't use it, so I didn't bother adding them in. I can add them in as a second PR, just so this one as the initial creation of the module doesn't change too much from PR creation time to submission time.

Reply:
I got the docs working. Creating PR for it.

Reply:
See https://github.com/angular/angular.js/pull/16945. You can find the working docs for $aria in the PR as a screenshot.